crates/sound/src/audio_backends/pipewire.rs

```diff
@@ -403,21 +403,22 @@ impl AudioBackend for PwBackend {
                         }
 
                         // to calculate as sizeof(int16_t) * NR_CHANNELS
-                        let frame_size = info.channels as u32 * size_of::<i16>() as u32;
+                        let frame_size = info.channels * size_of::<i16>() as u32;
```

Comment:
If this is not related, please move to another commit, since we may want to revert this commit, or keep this commit alone to work with the wrong Linux driver